nemo/collections/nlp/nm/trainables/common/megatron/megatron_bert_nm.py

```diff
@@ -90,7 +90,13 @@ def __init__(
             "vocab_file": vocab_file,
         }
 
-        initialize_megatron(None, megatron_args, ignore_unknown_args=True)
+        set_global_variables(extra_args_provider=None,
+                args_defaults=megatron_args,
+                ignore_unknown_args=True)
+        if self.factory._random_seed is None:
+            raise ValueError("Megatron Neural Module requires Neural Factory to have random_seed is not None.")
```

Comment:
should we maybe throw a warning but set self.factory._random_seed to some value and keep going?

Reply:
Done.